autoscale/src/main/resources/logback.xml

```diff
@@ -7,7 +7,7 @@
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
             <layout class="com.sequenceiq.cloudbreak.logger.MaskingPatternLayout">
-                <pattern>%date{ISO8601} [%thread] %method:%line %-5level %logger{5} - [type:%X{resourceType:-springLog}] [crn:%X{resourceCrn:-}] [name:%X{resourceName:-}] [tenant:%X{tenant:-}] [userCrn:%X{userCrn:-}] [requestid:%X{requestId:-}]  [traceId:%X{traceId:-}] [spanId:%X{spanId:-}] %msg%n</pattern>
+                <pattern>%date{ISO8601} [%thread] %method:%line %-5level %logger{5} - [type:%X{resourceType:-springLog}] [crn:%X{resourceCrn:-}] [name:%X{resourceName:-}] [tenant:%X{tenant:-}] [userCrn:%X{userCrn:-}] [requestid:%X{requestId:-}]  [traceId:%X{traceId:-}] [spanId:%X{spanId:-}] ::: %msg%n</pattern>
```

Comment:
Not very sure what log parsing you're referring to. If there's concerns, can revert this change.

While looking at logs manually though - this gives a much easier way to read logs, since the ::: can be used as a separator (otherwise real log lines are tough to find given the logged MDCContext is variable width, and several lines long when wrapped)

Reply:
fluentd used to have pattern for logs, but I do not know how it works nowadays? although I am also pretty sure that changing logging format can break log processing, but let's ask @oleewere

Reply:
This is not a must have change for the feature, better submit in a separate PR.

Reply:
Will revert this change.

Reply:
Fixed in the next push. Resolving conversation.